jerry-core/ecma/builtin-objects/ecma-builtin-regexp-prototype.cpp

```diff
@@ -71,7 +72,36 @@ ecma_builtin_regexp_prototype_exec (ecma_value_t this_arg, /**< this argument */
                     ecma_op_to_string (arg),
                     ret_value);
 
-    ret_value = ecma_regexp_exec_helper (obj_this, input_str_value, false);
+    ecma_property_t *bytecode_prop_p;
+    ecma_object_t *obj_p = ecma_get_object_from_completion_value (obj_this);
+    bytecode_prop_p = ecma_get_internal_property (obj_p, ECMA_INTERNAL_PROPERTY_REGEXP_BYTECODE);
+
+    if (!ECMA_GET_POINTER (void, bytecode_prop_p->u.internal_property.value))
```

Comment:
Please, add variable for the pointer, and change the condition to `var_name_p != NULL`.